cordova-js-src/platform.js

```diff
@@ -56,9 +57,10 @@ module.exports = {
             // OR cordova.require('cordova/platform').activationContext
             // activationContext:{type: actType, args: args};
             var activationHandler = function (e) {
-                var args = e.detail.arguments;
-                var actType = e.detail.type;
-                platform.activationContext = { type: actType, args: args };
+                // Making all the details available as activationContext
+                platform.activationContext = utils.clone(e.detail); 
+                platform.activationContext.args = e.detail.arguments;       /* for backwards compatibility */
+                platform.activationContext.actType = e.detail.type;         /* for backwards compatibility */
```

Comment:
I think you don't need an actType property for back-compat. The property (not the variable) was previously named just "type".